### ISSUE TITLE
TSLint v5.13.0 breaks linting

### ISSUE THREAD

**Issue by krotscheck** [angular/angular-cli#13783]
# 🐞 Bug report

### Command (mark with an `x`)
```
- [ ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [x] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?
Yes- Bug introduced via TSLint v5.13.0

### Description
All TSLint formatters now require that the linted file names be passed. This causes this line in devkit/build_angular/src/tslint/index to fail when using the checkstyle formatter (and likely others).
```
        if (!options.silent) {
          const Formatter = projectTslint.findFormatter(options.format);
          if (!Formatter) {
            throw new Error(`Invalid lint format "${options.format}".`);
          }
          const formatter = new Formatter();

          // FAILURE IS HERE.
          const output = formatter.format(result.failures, result.fixes);
          if(options.out) {

          }
          if (output) {
            this.context.logger.info(output);
          }
        }
```
Commit: https://github.com/palantir/tslint/commit/9000479b69c1f4808acf1dee284b47fbbbfbef13

## 🔬 Minimal Reproduction
In a current angular project, upgrade to `tslint v5.13.x`. Then run `ng lint --format=checkstyle`

## 🔥 Exception or Error
<pre><code>
Cannot read property 'map' of undefined
TypeError: Cannot read property 'map' of undefined
    at Formatter.format (/Users/mkrotscheck/Desktop/vmware/symphony/consumer-ui/node_modules/tslint/lib/formatters/checkstyleFormatter.js:42:40)
    at Observable.rxjs_1.Observable.obs [as _subscribe] (/Users/mkrotscheck/Desktop/vmware/symphony/consumer-ui/node_modules/@angular-devkit/build-angular/src/tslint/index.js:91:42)
    at Observable._trySubscribe (/Users/mkrotscheck/Desktop/vmware/symphony/consumer-ui/node_modules/rxjs/internal/Observable.js:44:25)
    at Observable.subscribe (/Users/mkrotscheck/Desktop/vmware/symphony/consumer-ui/node_modules/rxjs/internal/Observable.js:30:22)
    at /Users/mkrotscheck/Desktop/vmware/symphony/consumer-ui/node_modules/rxjs/internal/util/subscribeTo.js:22:31
    at Object.subscribeToResult (/Users/mkrotscheck/Desktop/vmware/symphony/consumer-ui/node_modules/rxjs/internal/util/subscribeToResult.js:10:45)
    at MergeMapSubscriber._innerSub (/Users/mkrotscheck/Desktop/vmware/symphony/consumer-ui/node_modules/rxjs/internal/operators/mergeMap.js:82:29)
    at MergeMapSubscriber._tryNext (/Users/mkrotscheck/Desktop/vmware/symphony/consumer-ui/node_modules/rxjs/internal/operators/mergeMap.js:76:14)
    at MergeMapSubscriber._next (/Users/mkrotscheck/Desktop/vmware/symphony/consumer-ui/node_modules/rxjs/internal/operators/mergeMap.js:59:18)
    at MergeMapSubscriber.Subscriber.next (/Users/mkrotscheck/Desktop/vmware/symphony/consumer-ui/node_modules/rxjs/internal/Subscriber.js:67:18)
</code></pre>

## 🌍 Your Environment
<pre><code>
Angular CLI: 6.2.9
Node: 10.15.0
OS: darwin x64
Angular: 6.1.10
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.13.3
@angular-devkit/build-angular      0.13.3
@angular-devkit/build-ng-packagr   0.13.3
@angular-devkit/build-optimizer    0.13.3
@angular-devkit/build-webpack      0.13.3
@angular-devkit/core               0.8.9
@angular-devkit/schematics         0.8.9
@angular/cli                       6.2.9
@ngtools/json-schema               1.1.0
@ngtools/webpack                   7.3.3
@schematics/angular                0.8.9
@schematics/update                 0.8.9
ng-packagr                         4.7.1
rxjs                               6.3.3
typescript                         2.9.1
webpack                            4.29.0
</code></pre>

**Comment by alan-agius4** [comment]
Tslint released 5.13.1 that fixes the breaking change.

**Comment by alan-agius4** [comment]
Tslint released 5.13.1 that fixes the breaking change.
